rmgpy/molecule/molecule.py

```diff
@@ -1618,4 +1618,21 @@ def toGroup(self):
         
         return group
 
-
+    def getNthNeighbor(self, startingAtoms, n, ignoreList = []):
```

Comment:
Remember to be very careful using an empty list as the default parameter to a function. I haven't checked this particular instance, but it is very rarely what you mean to do!  See http://docs.python-guide.org/en/latest/writing/gotchas/ and https://github.com/ReactionMechanismGenerator/RMG-Py/issues/640